parm/config/yaml/defaults.yaml

```diff
@@ -17,3 +17,8 @@ ocnanal:
   R2D2_OBS_DUMP: 's2s_v1'
   NICAS_RESOL: 1
   NICAS_GRID_SIZE: 15000
+
```

Comment:
isn't this already in this file (see up like 10 lines or so)

Reply:
I just added the following in this file: 

```
landanl:
  IO_LAYOUT_X: 1
  IO_LAYOUT_Y: 1
```

Reply:
Yeah I guess my question is, isn't this already in this file? Why is it here twice?

Reply:
The way the `defaults.yaml` is set, the key (in this case `landanl`) will change the templates in `config.landanl`. It is present in this file under a different key, so it will not impact `landanl`.

Reply:
It also allows `landanl` to have "different" layout, if they so desire.

Reply:
https://github.com/NOAA-EMC/global-workflow/blob/7b3e53ebf5f779aed652f11877935d507f3736ea/parm/config/yaml/defaults.yaml 

What is the difference between lines 5-7 and lines 21-23?

Reply:
doh. Yeah.  I did not notice that.  Thanks for clarifying.
You are right in this case.
These lines should be deleted.